src/renderer/wg_engine/tvgWgGeometry.cpp

```diff
@@ -87,13 +87,12 @@ void WgPolyline::appendPoint(WgPoint pt)
 }
 
 
-void WgPolyline::appendCubic(WgPoint p1, WgPoint p2, WgPoint p3)
+void WgPolyline::appendCubic(WgPoint p1, WgPoint p2, WgPoint p3, size_t nsegs)
 {
     WgPoint p0 = pts.count > 0 ? pts.last() : WgPoint(0.0f, 0.0f);
-    size_t segs = ((uint32_t)(p0.dist(p3) / 8.0f));
-    segs = segs == 0 ? 1 : segs;
-    for (size_t i = 1; i <= segs; i++) {
-        float t = i / (float)segs;
+    nsegs = nsegs == 0 ? 1 : nsegs;
+    for (size_t i = 1; i <= nsegs; i++) {
+        float t = i / (float)nsegs;
```

Comment:
to avoid repeated division I'd introduce dt = 1/nsegs outside the `for` loop and inside `t += dt` would be enough

Reply:
make a sence